### PR TITLE
Adding Orion Ack

### PR DESCRIPTION
**Pull request by jtaleric**
Adding the ack feature to avoid constant regression detection.

### CODE REVIEW

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-kubernetes-release-4.18-perfscale-control-plane-6nodes pull-ci-openshift-kubernetes-release-4.19-perfscale-control-plane-6nodes

File: ci-operator/step-registry/openshift-qe/orion/openshift-qe-orion-commands.sh [hunk 1]
```diff
@@ -42,6 +42,7 @@ if [ ${JUNIT} == true ]; then
   export EXTRA_FLAGS+=" --lookback ${LOOKBACK}d"
   export EXTRA_FLAGS+=" --output-format junit"
   export EXTRA_FLAGS+=" --save-output-path=junit.xml"
+  export ExTRA_FLAGS+=" --ack ack/${VERSION}_${ACK_FILE}"
```
Comment (afcollins) [inline ci-operator/step-registry/openshift-qe/orion/openshift-qe-orion-commands.sh:45]: typo (lowercase 'x') `ExTRA` vs. `EXTRA`

**Review by afcollins — APPROVED** [review]
/pj-rehearse ack

LGTM

Thanks!

**Pull request merged**